### ISSUE TITLE
Material transfers

### ISSUE THREAD

**Issue by hpema** [Aptronics/aptronics#17]
Hi Team,

Stock transfer/issues/receipts need to also have the logic for the batch selection. Currently batch screen popup does not show any batches.

The base rate should also update based on the lot selected and not allowed to be entered.

**Comment by SaiFi0102** [comment]
It seems to be working here. Can you elaborate the exact problem?

![Bathc Selection STE](https://user-images.githubusercontent.com/328330/68787768-f6c73200-0663-11ea-9c19-a092d21c6134.gif)

**Comment by hpema** [comment]
I think I was not adding quantity and was expecting to see the list of lots available...let me try again.

Also how do I do a transfer to assets?

The transfer to cost and write off is just a matter of selecting the account correctly. The users was looking for a quick button. We have only 2 types of transfers

to cost
to write off
to assets

is there a way to do as a quick entry?

The other option is maybe we adjust who does this and move it over to finance who understands accounts, cause currently sitting in procurement. Maybe we can combine this with a workflow where the user can add the stock details and then move over to finance for final approval and accounts selections before submitting. 

Thoughts?

**Comment by SaiFi0102** [comment]
Transfer of Assets is handled by "Asset Movement" Document: https://aptronics.agritheory.com/desk#Form/Asset%20Movement/New%20Asset%20Movement%201

**Comment by SaiFi0102** [comment]
About the Quick Entry and different types of transfers we can discuss it on call to get a better idea of what you're trying to achieve.
